js/modules/k6/http/request.go

```diff
@@ -357,70 +356,98 @@ func (h *HTTP) parseRequest(
 	return result, nil
 }
 
+func (h *HTTP) prepareBatchArray(
+	ctx context.Context, requests []interface{},
+) ([]httpext.BatchParsedHTTPRequest, []*Response, error) {
+	reqCount := len(requests)
+	batchReqs := make([]httpext.BatchParsedHTTPRequest, reqCount)
+	results := make([]*Response, reqCount)
+
+	for key, req := range requests {
```

Comment:
Is there any special meaning for this `key`? Looking at the `requests` type, I think it should be `i`.

Reply:
Resolved by https://github.com/loadimpact/k6/pull/1259/commits/bf560c1cfc8732991189fe64a93800b1db8af039